### ISSUE TITLE
 Tensorflow op [batch: QueueDequeueManyV2] is not supported

### ISSUE THREAD

**Issue by alex96295** [onnx/tensorflow-onnx#960]
**Describe the bug**

Hello, I am trying to convert a tf model to pytorch, I think there is an unsupported operation that prevents the conversion to happen completely

Do you have suggestions in how to solve it?

Here you find the checkpoint files, the .pb model generated from these checkpoint files after training (test.pb and test.pbtxt) and the model frozen for inference, mobilenetv2.pb --> http://www.mediafire.com/folder/bnohlzvks7h8r0y,hvwac3wm1iiwi4a,fkn4s0i4u7m93iw,hj7vo0v2afi3med,u54587g8g5vfs4g,flsf7a6z2vx223x,jrphzw6ypnv46pf/shared

The model I am converting is indeed **mobilenetv2.pb**

**Urgency**
If there are particular important use cases blocked by this or strict project-related timelines, please share more information and dates. If there are no hard deadlines, please specify none.

This thing is getting me stuck and perhaps it's not worth for what I have to do to waste too much time with a conversion, yet it is necesseray for me, so I would need it as soon as possible

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu
- Tensorflow Version: 1.15
- Python version: 3.7

**To Reproduce**
Describe steps/code to reproduce the behavior:

**Expected behavior**

I think, as in issue #806 that I have mentioned below, that the weird thing is this QueueDequeueManyV2 (which is related to the QueueDequeueV2 operation mentioned in that issue). I tried to follow your suggested approach there (modifying the same lines of controlflow.py you mentioned, adapting them to my problem), but it's not working in my particular case, or most probably I am missing something, since I am a beginner.

**Screenshots**

Error logs using the native onnx code:

![Cattura](https://user-images.githubusercontent.com/62642743/83938845-550e5a80-a7d8-11ea-95bb-baf106e45bbc.PNG)

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by guschmue** [comment]
We support QueueDequeueV2 but not QueueDequeueManyV2. @RandySheriffH, can we handle QueueDequeueManyV2 the same way like QueueDequeueV2 ?

**Comment by alex96295** [comment]
@guschmue Yes I had a look at the modifications done by @RandySheriffH to support QueueDequeueV2, but I would need an help to extend them towards QueueDequeueManyV2, since all the attempts I've done up to know... failed

Yet, thank you for the reply!

**Comment by RandySheriffH** [comment]
@alex96295 sure let me compose a fix.

**Comment by RandySheriffH** [comment]
https://github.com/onnx/tensorflow-onnx/pull/968, would u mind to try this out and let me know?

**Comment by RandySheriffH** [comment]
Note - you may need to set shape to your input:
python -m tf2onnx.convert --graphdef mobiletnetv2.pb --output converted.onnx --fold_const --opset 11 --inputs batch:0**[96,96,96,96]** --debug --verbose --outputs conv12/bias_add:0

**Comment by alex96295** [comment]
@RandySheriffH @guschmue the model was converted successfully, now I will have a try on it; 
Thank you a lot for your effort!

Could I just ask you why the 'batch:0' node input shape has been set to [96,96,96,96]? Is that the input of the model or something else? 
Because I know it takes as input images of dimension [batch_size, 224, 224, 3] and, during inference, I take the input image of dim [224, 224, 3], add a fake_batch dimension and thus provide the input as [1, 224, 224, 3]

**Comment by RandySheriffH** [comment]
@alex96295 
Yes please try latest onnxruntime release and let us know.
Then share us the inference script you are using along with the image if still found error.

**Comment by alex96295** [comment]
@RandySheriffH sorry for late reply
Thank you for the hint on the batches, I also had a try and did the same as you.
While for onnxruntime, the model is loaded correctly, I will see if the inference results is comparable to that of the model in tensorflow. Could there actually be some 'heavy' issues during conversion? Just for me to know
